src/_locales/ja/messages.json

```
@@ -260,69 +260,69 @@
+    "plugin_red_packet_description_incoming": "開封可能です",
+    "plugin_red_packet_description_fail": "作成失敗",
+    "plugin_red_packet_description_claimed": "既に受け取り済みです",
+    "plugin_red_packet_description_expired": "期限切れです",
+    "plugin_red_packet_description_claim_pending": "受け取り中",
+    "plugin_red_packet_description_empty": "期限切れです",
+    "plugin_red_packet_description_failover": "合計：{{total}} {{name}} / {{shares}} 株",
     "plugin_gitcoin_display_name": "プラグイン: Gitcoin Grant",
-    "plugin_gitcoin_readme": "このサービスを利用するで、寄附金額の 5% は <fund>Gitcoin grants development fund</fund> へ寄附しています。",
+    "plugin_gitcoin_readme": "このサービスを利用するにあたり、寄付金額の 5% が <fund>Gitcoin grants development fund</fund> に寄付されます",
     "plugin_gitcoin_readme_fund_link": "https://gitcoin.co/grants/86/gitcoin-sustainability-fund",
-    "plugin_gitcoin_not_valid": "無効入力",
-    "plugin_gitcoin_donate": "{{symbol}}{{amount}} {{type}} を寄附する",
-    "plugin_gitcoin_preview_card_fund": "この Grant 寄附する",
+    "plugin_gitcoin_not_valid": "無効",
+    "plugin_gitcoin_donate": "{{symbol}}{{amount}} {{type}} に寄付する",
+    "plugin_gitcoin_preview_card_fund": "この Grant に寄付する",
     "plugin_gitcoin_donate_amount": "数量",
-    "plugin_gitcoin_donated_successfully": "寄附成功",
-    "plugin_gitcoin_donated_failed": "寄附失敗",
+    "plugin_gitcoin_donated_successfully": "寄付成功",
+    "plugin_gitcoin_donated_failed": "寄付失敗",
     "plugin_gitcoin_dialog_share": "Share",
-    "plugin_file_service_display_name": "プラグイン: ファイル転送",
+    "plugin_file_service_display_name": "プラグイン: ファイル共有",
     "plugin_file_service_file_key_copied": "ファイルの鍵がコピーされました",
     "plugin_file_service_on_insert": "ファイルを使用する",
-    "plugin_file_service_uploading_preparing": "準備しています",
-    "plugin_file_service_uploading_estimating_time": "転送時間計算中...",
-    "plugin_file_service_uploading_in_minute_remaining": "{{seconds}}秒残っています",
-    "plugin_file_service_uploading_remaining": "{{minutes}}分{{seconds}}秒残っています",
+    "plugin_file_service_uploading_preparing": "準備中",
+    "plugin_file_service_uploading_estimating_time": "転送時間を計算中...",
+    "plugin_file_service_uploading_in_minute_remaining": "残り: {{seconds}}秒",
+    "plugin_file_service_uploading_remaining": "残り: {{minutes}}分{{seconds}}秒",
     "plugin_file_service_uploading_on_cancal": "アップロードしています...今このページを閉じないでください。",
     "plugin_file_service_recent_files": "最近のファイル",
-    "plugin_file_service_show_more": "他にも",
-    "plugin_file_service_file_key": "ファイルキー:",
+    "plugin_file_service_show_more": "更に",
+    "plugin_file_service_file_key": "File Key:",
     "plugin_file_service_unencrypted": "このファイルは暗号化されていません",
     "plugin_file_service_signing_failed": "サービスが使用できません。今しばらくお待ちください。",
-    "plugin_file_service_on_encrypt_it": "暗号化しよう",
+    "plugin_file_service_on_encrypt_it": "暗号化しましょう",
     "plugin_file_service_on_show_more": "サポートされていません",
     "plugin_file_service_legal_text": "このプラグインを使用することで<terms>利用規約</terms>並びに<policy>個人情報保護方針（プライバシーポリシー）</policy>に同意したこととなります。",
```

```suggestion
    "plugin_file_service_legal_text": "このプラグインを使用することで<terms>利用規約</terms>並びに<policy>個人情報保護方針（プライバシーポリシー）</policy>に同意したこととなります。",
```

`<terms>利用規約</terms>` and `<policy>個人情報保護方針（プライバシーポリシー）</policy>`

@Tedko keep original text?

https://legal.maskbook.com/arweave/file-service/

File Service not include other (non English) language legal file.

---

CC @Tedko @neruthes

---

Both work

---

<ruby>Privacy<rt>プライバシー</rt> Policy<rt>ポリシー</rt></ruby>